info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-07-28T18:56:11.045Z"}
info: Connected to the database. {"timestamp":"2022-07-28T18:56:11.127Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-07-28T18:58:11.870Z"}
info: Connected to the database. {"timestamp":"2022-07-28T18:58:11.927Z"}
error: 
    status - 500 
    message - Error: password not match
    at _callee2$ (H:\NodeJS\fundooNotes\fundooNotes\src\services\/user.service.js:33:11)
    at tryCatch (H:\NodeJS\fundooNotes\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (H:\NodeJS\fundooNotes\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (H:\NodeJS\fundooNotes\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (H:\NodeJS\fundooNotes\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\NodeJS\fundooNotes\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-28T19:00:37.108Z"}
